lecturer/junit_intro/src/main/java/by/jrr/start/Calculator.java

```diff
@@ -4,12 +4,12 @@
 
     private int result;
 
-    public int sum(int a, int b) {
-        return a + b;
+    public int multiplikation(int a, int b) {
+        return a * b;
     }
 
-    public int subs(int a, int b) {
-        return a - b;
+    public int division(int a, int b) {
+        return a/b;
```

Comment:
use sonar analyse.
// a/b 
this need to be surrounded with spaces.

Reply:
Sonar tests failed.